home/tests/test_content_import_export.py

```diff
@@ -710,17 +710,17 @@ def test_no_translation_key_nondefault(self, csv_impexp: ImportExport) -> None:
 
         assert e.value.row_num == 4
         # FIXME: Find a better way to represent this.
-        assert "translation_key" in e.value.message
-        assert "“” is not a valid UUID." in e.value.message
+        assert "translation_key" in e.value.message[0]
+        assert "“” is not a valid UUID." in e.value.message[0]
```

Comment:
Instead of just looking at the first message, let's rather assert on the whole list. If that's impractical (perhaps if there's a substring we don't know ahead of time) we can extract it with `[msg] = e.value.message` to make sure there's only one.